installer/assets/install

```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
```

Comment:
Can we use `set -eu` so that errors and unset variables abort the script? Should simplify error handling too.

Reply:
That's a good point. Thanks, Chris!